packages/squidex/schema/schemas/calendars.json

```diff
@@ -35,11 +35,12 @@
         "properties": {
           "fieldType": "String",
           "pattern": "^[a-zA-Z0-9.!#$%&’*+\\/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:.[a-zA-Z0-9-]+)*$",
+          "minLength": 8,
```

Comment:
Do we need min length for calendars?

Reply:
we can set this to 1, TBH, we just dont want empty strings

Reply:
yeah so they wouldnt put an empty string or something. You would think that wouldn't be possible with the regex but well, squidex